pandas/tests/arrays/categorical/test_analytics.py

```diff
@@ -35,6 +35,16 @@ def test_min_max(self):
         assert _min == "d"
         assert _max == "a"
 
+    def test_min_max_empty(self):
```

Comment:
can you paremeterize on ordered=[True, False]

Reply:
`ordered=False` would trigger a TypeError, as is tested in the `test_min_max` above it. Would you want it to return `NA` even if `ordered=False`?

Reply:
ahh right, ok, nvm on that then (maybe edit the test names to make this clear)